xml/signals/CTC-MB/.cvsignore

```diff
@@ -0,0 +1 @@
+.DS_Store
```

Comment:
We don’t use CVS in JMRI, this file should be removed.

Reply:
Deleted file.